### DISCUSSION TITLE
please hep me to setup rust desk flutter_briddge assignment

### DISCUSSION THREAD

**Discussion by iamparas0** [community/community#58394] [GitHub Education]
### Hi everyone,

i need help to solve this assignment please help me gow to do

**Comment by samus-aran** [comment]
Hi @iamparas0 ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
